crates/packager/src/lib.rs

```diff
@@ -101,6 +101,35 @@ pub use sign::SigningConfig;
 
 pub use package::{package, PackageOuput};
 
+fn parse_log_level(verbose: u8) -> tracing::Level {
+    match verbose {
+        0 => tracing_subscriber::EnvFilter::builder()
+            .from_env_lossy()
+            .max_level_hint()
+            .and_then(|l| l.into_level())
+            .unwrap_or(tracing::Level::INFO),
+        1 => tracing::Level::DEBUG,
+        2.. => tracing::Level::TRACE,
+    }
+}
+
+/// Inits the tracing subscriber.
+pub fn init_tracing_subscriber(verbosity: u8) {
```

Comment:
I would like to keep the tracing_subscriber out of the library, the node CLI could either depend directly on it or we add this behind a feature flag that is activated on the node CLI